.betterer.results

```diff
@@ -6136,43 +6136,6 @@ exports[`better eslint`] = {
     "public/app/plugins/datasource/elasticsearch/hooks/useStatelessReducer.ts:5381": [
       [0, 0, 0, "Do not use any type assertions.", "0"]
     ],
-    "public/app/plugins/datasource/elasticsearch/language_provider.ts:5381": [
-      [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
-      [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
-      [0, 0, 0, "Unexpected any. Specify a different type.", "2"],
```

Comment:
im going to fix this and ask you to rebase after

Reply:
please rebase once this is merged https://github.com/grafana/grafana/pull/55682/files ❤️